db/controllers/controller.js

```diff
@@ -96,3 +97,9 @@ exports.deleteCommentByID = (req, res, next) => {
     })
     .catch((err) => next(err));
 };
+
+exports.getAllUsers = (req, res, next) => {
+  fetchAllUsers()
+    .then((users) => res.status(200).send(users))
```

Comment:
data should always be returned inside an object and on an appropriate key, example: `.send({ users: users });`